update-lb4-docs.sh

```diff
@@ -0,0 +1,19 @@
+org=strongloop
+repo=loopback-next
+target=pages/en/lb4
+branch=move-docs-into-monorepo
+tmp=$target/tempdir
+
+mkdir -p $tmp
+# pull down all the markdown files (reserving dir structure)
+# given the GitHub org, repo, and branch
+curl -L https://github.com/$org/$repo/archive/$branch.tar.gz \
+   | tar --strip-components 1 -C $tmp -xzf- '*.md'
+cp -R $tmp/packages $target/readmes/$repo
+#rename relative link extensions from md to html
+sed -E -i 's/\(([^(]+)\.md/(\1.html)/g' $tmp/docs/*.md
```

Comment:
You have:
```
s/\(([^(]+)\.md/(\1.html)/g
```
It should be this:
```
s/\(([^(]+)\.md\)/(\1.html)/g
```
You need to match the closing `)` literal otherwise it doesn't get replaced by the `)` literal in the replacement and you end up with double `))` on the output - and also incorrectly matching:
 `(index.md other stuff)` -> `(index.html) other stuff)`

Reply:
Good catch. Actually I wanted to include the cases where `.md` is followed by `#` and left out the literal match for `)`. I will remove it from the replacement as well.